### PR TITLE
Weekly merge

### PR DESCRIPTION
**Pull request by connorwstein**
## Motivation
Keep our fork in sync.

## Solution
- Only real change is evm utils were extracted to core/chains/evm/utils

### CODE REVIEW

File: core/chains/evm/logpoller/log_poller_test.go [hunk 1]
```diff
@@ -1295,7 +1296,7 @@ func TestLogPoller_DBErrorHandling(t *testing.T) {
 	require.ErrorContains(t, err, "Invalid replay block number")
 
 	// Force a db error while loading the filters (tx aborted, already rolled back)
-	require.Error(t, utils.JustError(db.Exec(`invalid query`)))
+	require.Error(t, commonutils.JustError(db.Exec(`invalid query`)))
```
Comment (connorwstein) [inline core/chains/evm/logpoller/log_poller_test.go:1299]: Small change here

File: core/chains/evm/logpoller/observability_test.go [hunk 2]
```diff
@@ -16,12 +16,11 @@ import (
 	"github.com/prometheus/client_golang/prometheus/testutil"
 
 	"github.com/smartcontractkit/chainlink-common/pkg/logger"
-
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/utils"
```
Comment (connorwstein) [inline core/chains/evm/logpoller/observability_test.go:19]: Ditto

File: integration-tests/types/config/node/core.go [hunk 3]
```diff
@@ -23,7 +25,6 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/keys/ethkey"
 	"github.com/smartcontractkit/chainlink/v2/core/store/models"
 	"github.com/smartcontractkit/chainlink/v2/core/utils"
-	"github.com/smartcontractkit/chainlink/v2/core/utils/config"
```
Comment (connorwstein) [inline integration-tests/types/config/node/core.go:26]: @AnieeG would be good to double check this, was a conflict

Comment (AnieeG) [reply integration-tests/types/config/node/core.go:26]: looks right.

File: integration-tests/go.mod [hunk 4]
```diff
@@ -1,19 +1,19 @@
-module github.com/smartcontractkit/ccip/integration-tests
+module github.com/smartcontractkit/chainlink/integration-tests
```
Comment (AnieeG) [inline integration-tests/go.mod]: we need to change it back to `ccip`.

File: integration-tests/go.mod [hunk 5]
```diff
@@ -497,8 +488,4 @@ replace (
 
 	// until merged upstream: https://github.com/mwitkow/grpc-proxy/pull/69
 	github.com/mwitkow/grpc-proxy => github.com/smartcontractkit/grpc-proxy v0.0.0-20230731113816-f1be6620749f
-
-	github.com/smartcontractkit/chainlink/integration-tests => ../integration-tests
```
Comment (AnieeG) [inline integration-tests/go.mod]: This is required as well

**Review by AnieeG — CHANGES_REQUESTED** [review]
`integration-tests/go.mod` needs changes

**Review by AnieeG — APPROVED** [review]

**Review by makramkd — APPROVED** [review]

**Review by RensR — APPROVED** [review]

**Pull request merged**